### ISSUE TITLE
NTR: Increased fecal coproporphyrin 3

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#5685]
**Preferred term label:**  
Increased fecal coproporphyrin 3

**Synonyms**  
Elevated fecal coproporphyrin 3; High fecal coproporphyrin 3; Elevated stool coproporphyrin 3; High stool coproporphyrin 3

**Definition (free text, please give PubMed ID)**
Abnormally high concentration of coproporphyrin 3 in feces

[PMID:3327428](http://www.ncbi.nlm.nih.gov/pubmed/3327428)

**Parent term (use hpo.jax.org/app)**
See issue #5627 - Increased fetal porphyrin

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)** 
- [121300](http://omim.org/entry/121300) Hereditary coproporphyria
- [176000](http://omim.org/entry/176000) Acute intermittent porphyria  

[PMID:10905759](http://www.ncbi.nlm.nih.gov/pubmed/10905759)

**Your nano-attribution (ORCID)**  
https://orcid.org/0000-0002-8169-9049

**Comment by callahantiff** [comment]
Thanks @pnrobinson!
